### DISCUSSION TITLE
<Copy as cURL>

### DISCUSSION THREAD

**Discussion by khansami213** [community/community#133390] [Programming Help]
### Select Topic Area

Question

### Body

![Screenshot 2024-07-24 183300](https://github.com/user-attachments/assets/aa4dfbe2-4f3b-4c31-8657-ede3e04229c7)
I want to scrap data from one site using PHP cURL to fetch headers i neesd to copy as cURL in developer tool but i dont option "copy as cURL" in my window how to enbale that.

**Comment by LiteBrite82** [comment]
Thanks for posting in the GitHub Community, @khansami213 ! 

We’ve moved your post to our Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.

**Comment by ebndev** [comment]
Hi @khansami213!

I am closing this discussions as it is a duplicate of [this one](https://github.com/orgs/community/discussions/133391) - hopefully other users will be able to help you there 😄
